### ISSUE TITLE
for some reason it stopped loading

### ISSUE THREAD

**Issue by lezlikov** [htmltiger/config-editor-card#47]
![image](https://user-images.githubusercontent.com/73757896/197002010-4f8bf3ec-c362-4dc9-900f-64d2ec951c83.png)
![image](https://user-images.githubusercontent.com/73757896/197002142-37e0c4fa-8769-41b2-b7de-e3d729813e63.png)
![image](https://user-images.githubusercontent.com/73757896/197002218-29755581-3307-4cdb-a5a7-b66aca3d4c2c.png)

What can I do?

**Comment by htmltiger** [comment]
check `home-assistant.log` in your config directory and post any details that we need to know.
it could also be file permission issue of the file you are editing.

**Comment by lezlikov** [comment]
2022-10-20 19:22:22.844 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-10-20 19:22:22.850 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration config_editor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-10-20 19:22:22.853 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration ble_monitor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-10-20 19:22:50.633 WARNING (MainThread) [homeassistant.setup] Setup of input_datetime is taking over 10 seconds.
2022-10-20 19:22:50.649 WARNING (MainThread) [homeassistant.setup] Setup of input_number is taking over 10 seconds.
2022-10-20 19:22:50.653 WARNING (MainThread) [homeassistant.setup] Setup of input_boolean is taking over 10 seconds.
2022-10-20 19:22:50.656 WARNING (MainThread) [homeassistant.setup] Setup of input_select is taking over 10 seconds.
2022-10-20 19:22:50.660 WARNING (MainThread) [homeassistant.setup] Setup of zone is taking over 10 seconds.
2022-10-20 19:22:50.663 WARNING (MainThread) [homeassistant.setup] Setup of input_text is taking over 10 seconds.
2022-10-20 19:22:50.666 WARNING (MainThread) [homeassistant.setup] Setup of schedule is taking over 10 seconds.
2022-10-20 19:22:50.668 WARNING (MainThread) [homeassistant.setup] Setup of timer is taking over 10 seconds.
2022-10-20 19:22:50.670 WARNING (MainThread) [homeassistant.setup] Setup of counter is taking over 10 seconds.
2022-10-20 19:22:50.671 WARNING (MainThread) [homeassistant.setup] Setup of input_button is taking over 10 seconds.
2022-10-20 19:22:50.673 WARNING (MainThread) [homeassistant.setup] Setup of application_credentials is taking over 10 seconds.
2022-10-20 19:22:50.675 WARNING (MainThread) [homeassistant.setup] Setup of script is taking over 10 seconds.
2022-10-20 19:22:50.678 WARNING (MainThread) [homeassistant.setup] Setup of automation is taking over 10 seconds.
2022-10-20 19:23:08.641 WARNING (SyncWorker_1) [pyroute2.netlink.rtnl.tcmsg.common] tcmsg: [Errno 2] No such file or directory: '/proc/net/psched'
2022-10-20 19:23:08.671 WARNING (SyncWorker_1) [pyroute2.netlink.rtnl.tcmsg.common] the tc subsystem functionality is limited

**Comment by RandomGithubUserX** [comment]
Hi htmltiger,

Thanks for the great work first of all.

Since a year i do have a disturbing issue with the dashboard as it does not load any n home assistant apk and on different browsers, just showing this:
![Screenshot_2022-12-09-09-49-26-43_c3a231c25ed346e59462e84656a70e50.jpg](https://user-images.githubusercontent.com/97596205/206662669-0cae235c-aa02-436b-816a-850ad5502ecc.jpg)

First I reinstalled and tried several other ways to make it work, but at the end I have find out that it is related to the  browser/app cache locally. So for example it is always loading correctly in incognito mode in browser. 

Is there a way you can do about this in your next update please? In case I can assist just let me know. 

Br. 

Attila

**Comment by htmltiger** [comment]
Have you upgraded integration?
https://github.com/htmltiger/config-editor
If its working fine in incognito clear site cache and local storage

**Comment by RandomGithubUserX** [comment]
Of course I do have the latest version. Cache delete works only for one time than I get the same issue again... 
Forgot to mention that I am using Hasswp https://github.com/AlexxIT/HassWP , but don't believe that it is related to the issue.

**Comment by htmltiger** [comment]
Can you please check Developer tools > states > search for config_editor and post the results? 
the version should be 4 
![1s](https://user-images.githubusercontent.com/1429451/206863260-3d51a72c-4cc2-46b0-b213-6bd1f44201e2.png)

**Comment by RandomGithubUserX** [comment]
I have this there :

![Screenshot_2022-12-10-17-00-50-08_c3a231c25ed346e59462e84656a70e50.jpg](https://user-images.githubusercontent.com/97596205/206864014-8b92aaf1-407a-4fa0-b3b1-bcf9ed51e3dd.jpg)

**Comment by Looney1976** [comment]
Hello, I've tried to install the editor as well but the frontend won't load. I've tried it multiple times, two methods but to no avail.

HA log shows:
2022-12-14 09:00:28.541 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration config_editor which has not been tested by Home Assistant. This co
2022-12-14 09:00:28.542 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component m
2022-12-14 09:00:28.543 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration afvalbeheer which has not been tested by Home Assistant. This comp
~
~
~
